### PR TITLE
feat: add posts page and post-details page

### PR DESCRIPTION
**Pull request by Kr1shnendu**

### CODE REVIEW

File: firebaseConfig.ts
Comment (a2ke5e1) [inline firebaseConfig.ts:1]: Remove this file. Firebase Config file should be in .gitignore file

Comment (Kr1shnendu) [reply firebaseConfig.ts:1]: Done

File: src/app/layout.tsx
Comment (a2ke5e1) [inline src/app/layout.tsx:1]: Keep it as it is. It ensures enough margin in different devices

Comment (Kr1shnendu) [reply src/app/layout.tsx:1]: okay

**Pull request merged**